common/components/SettingsForm.tsx

```diff
@@ -120,6 +121,15 @@ const useSelectableSettingStyles = makeStyles<Theme>((theme) => ({
     },
 }));
 
+export const LabelWithHoverEffect = styled(FormControlLabel)(({ theme }) => ({
```

Comment:
Can you put this in its own file and export this function as the default export? Then import it as needed here and in `VideoDataSyncDialog.tsx`. This will allow just that component to be imported without importing the code in this file as well.